main.go

```diff
@@ -553,3 +552,26 @@ func abs(i int) int {
 	}
 	return i
 }
+
+func createClockEmoji(t time.Time, loc *time.Location) string {
+	cHour := t.In(loc).Hour()
+	cMinutes := t.In(loc).Minute()
+	clocktext := ":clock"
+	if cMinutes >= 45 {
```

Comment:
This whole cMinute rounding thing could and should be in a `switch` statement. Go switch statements can be used without a value (`switch {`) to instead match on booleans.